libs/arcade-cli/arcade_cli/main.py

```diff
@@ -219,12 +224,28 @@ def show(
         "--no-tls",
         help="Whether to disable TLS for the connection to the Arcade Engine.",
     ),
+    worker: bool = typer.Option(
+        False,
+        "--worker",
+        "-w",
+        help="Show full worker response structure including error, logs, and authorization fields.",
```

Comment:
could be helpful to mention that this only makes a difference when `-t` flag is provided. Maybe show a warning if `--worker` is provided, but not `-t`? Took me too long to realize this